### PR TITLE
Feat/chat bot

### PR DESCRIPTION
**Pull request by tung-lee**
From weminal with luv ❤️

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,58 +27,40 @@ OWallet is developed based on Keplr extension and currently maintained by Oraich
...
- *  https://github.com/chainapsis/keplr-wallet/tree/0e137373ac4f526caf97b4694de47fe1ba543bd8
- */
-```
-Full text: [LICENSE.txt](LICENSE.txt)
\ No newline at end of file
+    `yarn build`
+    
+    When building for the first time, you may encouter the error. Do not worry, try to run `yarn build` again.
+    
+    And for the last package @owallet/extension, you should go directly into that package then `yarn build`
```
Comment (tucq88) [inline readme.md:66]: Love how you clean it but I'd suggest to keep Contribution & Release sections. Especially the license is also important, we should keep it short but must available.

File: packages/extension/src/pages/chatbot/index.tsx [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+          },
+          body: JSON.stringify({
+            user_address: userAddr,
+            user_input: prompt,
+            pair_contract: pairContractAddr,
+          }),
+        });
+        const data = await resp.json();
+        setIsLoading(false);
+        console.log(data);
```
Comment (tucq88) [inline packages/extension/src/pages/chatbot/index.tsx:133]: How about remove all `console.log` for production code?

File: packages/extension/src/pages/chatbot/index.tsx [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div ref={messagesEndRef} />
+      </div>
+    </HeaderLayout>
+  );
+});
```
Comment (tucq88) [inline packages/extension/src/pages/chatbot/index.tsx:361]: The file is too long with various components & responsibilities. I'd suggest lean it with services & splitting components. Keep single file less than 100 LoC is recommended (shorter is better)

File: packages/extension/src/pages/chatbot/index.tsx [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+                        payload: (evt.target as HTMLDivElement).innerText,
+                      })
+                    }
+                  >
+                    <p>Compare the price of Airight on Oraidex</p>
+                  </div>
+                </div>
+              )}
+
+              {(status === StatusEnum.CHAT || messages.length != 0) && (
```
Comment (tucq88) [inline packages/extension/src/pages/chatbot/index.tsx:278]: Let's be consistent and use `!==`

File: packages/extension/src/pages/chatbot/index.tsx [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+                        pairContractAddr,
+                        chosenOption,
+                        setIsLoading
+                      )
+                    }
+                    // onClick={() => testChatUI()}
+                    style={{ cursor: 'pointer' }}
+                    className="arrow-up-square"
+                    alt="Arrow up square"
+                    src={require('../../public/assets/img/arrow-up-square.svg')}
```
Comment (tucq88) [inline packages/extension/src/pages/chatbot/index.tsx:333]: Let's not import with direct `require` like this. It's gonna hard to replace in future

File: packages/extension/src/pages/chatbot/index.tsx [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+
+const BACKEND_URL = 'https://oraidex-tools.fly.dev';
+
+const initialState = {
+  messages: JSON.parse(localStorage.getItem('messages')) || [],
+  prompt: '',
+  status: StatusEnum.READY,
+  chosenOption: OptionEnum.ORAIDEX,
+  pairContractAddr:
+    'orai1agqfdtyd9lr0ntmfjtzl4f6gyswpeq4z4mdnq4npdxdc99tcw35qesmr9v',
```
Comment (tucq88) [inline packages/extension/src/pages/chatbot/index.tsx:27]: Can you justify a bit with this adddress & BACKEND_URL above? 
How they gonna be maintained?

Comment (tung-lee) [reply packages/extension/src/pages/chatbot/index.tsx:27]: In the Oraichain AI X DeFi Cook-off hackathon, we have implemented a new feature for Owallet, which is currently in its MVP version. For maintenance purposes, we are able to provide you with the source code of the backend. Additionally, we have set up a pair contract address specifically for the purpose of demo swapping between the native token and CW20 token. Please note that the pair contract address is currently hard-coded, but we have plans to improve this in the future. You can find detailed information about our build here [https://dorahacks.io/buidl/9242](https://dorahacks.io/buidl/9242?fbclid=IwAR28gezzfOgvoxxfaO-x0Oi6deScsN5PjN-2bqJiLZATYF6K0poyJ7t8PaM)

**Pull request closed without merging**